docs/README.md

```diff
@@ -35,75 +35,41 @@ Running this simulation requires two computers communicating over the network: a
 
 The `robot_studio_resources` directory contains a Pack and Go file for RobotStudio, which packages the station with the virtual controller and add-in configuration. To set up the robot using the packaged solution:
 
-1. Download `IRB1200_5_90.rspag` onto the computer with RobotStudio
+1. Download `IRB1200_5_90.rspag` (or `IRB1200_5_90_EXTAX.rspag` for an example with an external axis) onto the computer with RobotStudio
```

Comment:
Quick note before I forget:  please add `colcon build` to the build section near the top of this file

Reply:
I went ahead and pushed a small commit for this. LGTM! Splitting the docs into smaller chunks was a good idea, I think